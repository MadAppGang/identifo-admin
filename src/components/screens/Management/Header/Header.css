.iap-management-header {
  width: 100%;
  height: 70px;
  border-bottom: 1px solid #eee;
  display: flex;
  align-items: center;
  z-index: 999;
  position: fixed;
  background-color: rgba(255, 255, 255, .95);
}

.iap-management-header__inner {
  display: flex;
  align-items: center;
  justify-content: space-between;
  width: 100%;
}

.iap-management-header__logo {
  display: flex;
  padding: 0;
}

.iap-management-header__logo span {
  font: 400 18px 'Circe';
  display: inline-block;
  display: flex;
  align-items: center;
}

.iap-management-header__logo span:first-of-type {
  padding-right: 9px;
  border-right: 1px solid #8d8d8d;
  color: #3d3d3d;
  height: 18px;
}

.iap-management-header__logo span:last-of-type {
  font: 400 14px 'Circe';
  padding-left: 9px;
  color: #aaa;
  height: 20px;
}