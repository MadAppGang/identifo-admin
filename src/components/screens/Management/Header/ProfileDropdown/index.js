import React from 'react';
import DropdownIcon from '~/components/icons/DropdownIcon';
import LogoutSection from './LogoutSection';
import Dropdown from '~/components/shared/Dropdown';
import './Profile.css';

const ProfileDropdown = () => {
  return (
    <Dropdown>
      {({ isOpen, open }) => (
        <div className="iap-header-profile">
          <button
            type="button"
            className="iap-header-profile__trigger"
            onClick={open}
          >
            <span>Admin Panel</span>
            <DropdownIcon />
          </button>
          {isOpen && (
            <div className="iap-profile-dropdown">
              <LogoutSection />
            </div>
          )}
        </div>
      )}
    </Dropdown>
  );
};

export default ProfileDropdown;
