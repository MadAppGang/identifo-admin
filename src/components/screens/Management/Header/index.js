import React from 'react';
import './Header.css';

const ManagementScreenHeader = () => {
  return (
    <header className="iap-management-header">
      Hello, world!
    </header>
  );
};

export default ManagementScreenHeader;
