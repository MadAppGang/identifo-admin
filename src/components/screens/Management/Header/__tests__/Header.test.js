import Header from '..';

describe('<ManagementScreenHeader />', () => {
  test('renders as expected', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
