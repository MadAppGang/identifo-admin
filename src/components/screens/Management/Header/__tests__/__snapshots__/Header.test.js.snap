// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ManagementScreenHeader /> renders as expected 1`] = `
<header
  className="iap-management-header"
>
  <Container>
    <span
      className="iap-management-header__logo"
    >
      Identifo
    </span>
  </Container>
</header>
`;
